stories/dropZone.stories.tsx

```diff
@@ -7,6 +7,8 @@ export default {
   title: 'Layout/DropZone',
   args: {
     color: 'black',
+    // eslint-disable-next-line no-alert
+    onDrop: () => alert('files uploaded'),
```

Comment:
Is there a possibility to display the list of dropped files ?

Like in [this example](https://github.com/cheminfo/filelist-utils/blob/59e182c6649ebcf39df7880b47482190b18ac2b3/html/index.html).

Also could we display and edit the 'onDrop' callback in the storybook ?

Reply:
> Is there a possibility to display the list of dropped files ?

yeah it's possible

Reply:
> Is there a possibility to display the list of dropped files ?

table created.



> Also could we display and edit the 'onDrop' callback in the storybook ?

yes i added an onDrop action in the story